api/output_policy.go

```diff
@@ -59,17 +75,15 @@ func (d *OutputPolicyError) buildSamplePolicy() (string, error) {
 		capabilities = append(capabilities, "list")
 	}
 
-	// sanitize, then trim the Vault address and v1 from the front of the path
-	path, err := url.PathUnescape(d.path)
-	if err != nil {
-		return "", fmt.Errorf("failed to unescape request URL characters: %v", err)
-	}
-
 	// determine whether to add sudo capability
-	if IsSudoPath(path) {
+	if IsSudoPath(d.path) {
 		capabilities = append(capabilities, "sudo")
 	}
 
+	return fmtOutput(d.path, capabilities), nil
+}
+
+func fmtOutput(path string, capabilities []string) string {
```

Comment:
I would name this something more specific to the output policy, e.g. formatOuputPolicy (since this function is visible within the entire `vault/api` package).

Alternatively, you could make it a method of OuputPolicyError.

In either case, I'd replace "fmt" with "format", which may be friendlier for non-native speakers.